starter-code/src/components/SearchBar.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
+
+class SearchBar extends Component {
+  state = {
+    search: "",
+    checkbox: false
+  };
+
+  handleChange = event => {
+    // let { value } = event.target;
+    // this.props.updateString(value);
+    this.setState({ search: event.target.value });
+    this.props.updateString(event.target.value);
+  };
+
+  handleCheckbox = event => {
+    this.setState({ checkbox: event.target.checked });
+    this.props.updateCheckBox(event.target.checked);
+  };
+
+  render() {
+    return (
+      <div className="SearchBar">
+        <form>
+          <div className="form-group">
+            <label htmlFor="searchbar">Search</label>
+            <input
+              type="text"
+              className="form-control"
+              id="searchbar"
+              name="searchString"
+              value={this.props.searchString}
```

Comment:
Cette props n'existe pas, il faudrait plutôt avoir `value={this.state.searchString}`